lib/tasks/case_contact_draft_cleanup.rake

```diff
@@ -0,0 +1,6 @@
+desc "delete draft case contacts marked for clean up, run by heroku scheduler."
```

Comment:
good desc

Reply:
for later PR thought - we may want to move away from heroku scheduler towards something like 

```
      include Sidekiq::Job
      sidekiq_options queue: :general_12h

      def self.crontab
        "0 8 * * *"
      end
```

Reply:
I'm not that familiar with heroku scheduler but would like to learn so I'm curious, what's the rationale for moving away from it in favor of Sidekiq?